contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -51,18 +51,22 @@
         }"
       >
         <VLayout row class="px-3">
-          <IconButton
+          <KRouterLink>
+          <KIcon
             icon="collapseAll"
             :text="$tr('collapseAllButton')"
             @click="collapseAll"
           />
+          </KRouterLink>
           <VSpacer />
-          <IconButton
+          <KRouterLink>
+          <KIcon
```

Comment:
Since `KIcon` doesn't take the prop `:disabled`, `KIconButton` might be a better replacement here.

Reply:
Okay! Done.